util/network_linux.go

```diff
@@ -0,0 +1,136 @@
+// +build linux
+
+package util
```

Comment:
I might be a stickler here (so feel free to ignore this one!) but this might make sense in a different package like `net`. `util` is notoriously generic, especially if this is extended. Again not a huge deal if it's left as-is

Reply:
Agreed -- I originally did it because we have `util.containers`. Moved.